contrib/ruby/ext/trilogy-ruby/cext.c

```diff
@@ -811,10 +811,12 @@ static VALUE read_query_response(VALUE vargs)
             }
         }
 
+        rb_encoding *conn_enc = rb_to_encoding(ctx->encoding);
+
 #ifdef HAVE_RB_INTERNED_STR
```

Comment:
This should be updated to `HAVE_RB_ENC_INTERNED_STR` and `extconf.rb` should also be updated accordingly.

Reply:
Thanks, I changed this to use `HAVE_RB_ENC_INTERNED_STR` and added the corresponding `have_func` to `extconf.rb`